Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,124 @@
+## 195: Accessibility improvements in Windows Forms controls
```

Comment:
Are we going to abandon/close #477 then?

Reply:
Yep. I believe that is the plan.

Reply:
done